src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -52,8 +55,31 @@
  */
 public class ManualCondition extends PromotionCondition {
     private String users;
+        
     private List<ParameterDefinition> parameterDefinitions = new ArrayList<ParameterDefinition>();
     
+    
+    //Code to fetch Approve Promotion Button label from properties file
+    private String executeButtonLabel;
+    
+    public String getExecuteButtonLabel() {
+    	try {
+    		//to use Custom label, add deamon property 'PROMOTION_BUILD_APPROVE_BUTTON_CUSTOM_LABEL' to "/etc/init.d/jenkins" file in value to JAVA_CMD
+    		//sample: JAVA_CMD="$JENKINS_JAVA_CMD $JENKINS_JAVA_OPTIONS -DJENKINS_HOME=$JENKINS_HOME -DPROMOTION_BUILD_APPROVE_BUTTON_CUSTOM_LABEL=Execute -jar $JENKINS_WAR"
+    		executeButtonLabel = System.getProperty("PROMOTION_BUILD_APPROVE_BUTTON_CUSTOM_LABEL");
```

Comment:
In Jenkins a common practice is to include class name into the beginning of he property name. Just see examples in this plugin